### PR TITLE
Use AddExaminationButton in main_page_empty

### PR DESCRIPTION
**Pull request by s174508**

### CODE REVIEW

File: lib/ui/main_page_empty.dart [hunk 1]
```diff
@@ -32,25 +33,15 @@ class MainPageEmptyResults extends StatelessWidget {
               textAlign: TextAlign.center,
             ),
             verticalSpacing(48),
-            GestureDetector(
```
Comment (jeyjey626) [inline lib/ui/main_page_empty.dart:35]: The gesture detector makes it so the examination starts also when they click the test. I don't think it's necessary to remove rhat

**Review by jeyjey626 — COMMENTED** [review]
Left comments in code

File: lib/ui/main_page_empty.dart (hunk 1 above)
Comment (s174508) [reply lib/ui/main_page_empty.dart:35]: In that case, I assume the sliding animation should be applied to the gesture detector as well?

Comment (jeyjey626) [reply lib/ui/main_page_empty.dart:35]: Yes, that would be the case. I forgot about this use case :)

**Review by jeyjey626 — APPROVED** [review]

**Pull request merged**